### PR TITLE
Samsung Internet 15.0 Features

### PR DESCRIPTION
**Pull request by AdaRoseCannon**
#### Summary
Adds features for Samsung Internet 15.0 

#### Test results and supporting details

Samsung Internet 15.0 will be based on M90 although no WebNFC support yet.

This is generated using the mirror script and then corrected by hand.

### CODE REVIEW

File: api/ProgressEvent.json [hunk 1]
```diff
@@ -142,7 +142,8 @@
               "version_removed": "6"
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": true,
+              "version_removed": "1.0"
```
Comment (vinyldarkscratch) [inline api/ProgressEvent.json:146]: This change is really weird.  Did this come from the mirroring script?  If so, can you file an issue?

File: html/global_attributes.json [hunk 2]
```diff
@@ -176,18 +176,18 @@
             },
             "samsunginternet_android": [
               {
+                "version_added": "9.0",
                 "notes": [
                   "Support added for <code>autocomplete</code> on the <code>&lt;textarea&gt;</code> and <code>&lt;select&gt;</code> elements.",
-                  "Samsung Internet does not accept <code>off</code> as a value.  See <a href='https://crbug.com/587466'>bug 587466</a>."
-                ],
-                "version_added": "9.0"
+                  "Chrome does not accept <code>off</code> as a value.  See <a href='https://crbug.com/587466'>bug 587466</a>."
```
Comment (vinyldarkscratch) [inline html/global_attributes.json:182]: This change seems to just be a potential issue with the notes conversion in the mirroring script.  Can we revert this change?

File: html/global_attributes.json [hunk 3]
```diff
@@ -176,18 +176,18 @@
...
+                ]
               },
               {
+                "version_added": true,
                 "notes": [
                   "Originally only supported on the <code>&lt;input&gt;</code> element.",
-                  "Samsung Internet does not accept <code>off</code> as a value.  See <a href='https://crbug.com/587466'>bug 587466</a>."
-                ],
-                "version_added": true
+                  "Chrome does not accept <code>off</code> as a value.  See <a href='https://crbug.com/587466'>bug 587466</a>."
```
Comment (vinyldarkscratch) [inline html/global_attributes.json:189]: Also seems to be a script issue here.  Can we revert this too?

**Review by vinyldarkscratch — DISMISSED** [review]
Thanks for adding Samsung Internet 15.0!  I've run through and everything looks good to me, aside from a few things.  These seem to be potential issues with the mirroring script more than anything else, I'm working to fix up the mirroring script ASAP!

**Comment by AdaRoseCannon** [comment]
Thank you so much for finding these I'd missed them in my pass, yes they are all mirror script errors. I'll file an issue.

File: css/selectors/is.json [hunk 4]
```diff
@@ -220,7 +220,7 @@
                 "version_added": false
               },
               "samsunginternet_android": {
-                "version_added": false
+                "version_added": "15.0"
```
Comment (lukewarlow) [inline css/selectors/is.json:223]: The mirroring script appears to have missed the top section of this file. Currently, Samsung Internet isn't marked as supporting :is().

**Comment by AdaRoseCannon** [comment]
Changes fixed, thank you!

File: api/Window.json [hunk 5]
```diff
@@ -6004,7 +6004,7 @@
               "version_added": false
             },
             "samsunginternet_android": {
-              "version_added": false
+              "version_added": "15.0"
```
Comment (foolip) [inline api/Window.json:6007]: Excellent, this is the last remaining bit of https://github.com/mdn/browser-compat-data/pull/9666 (after https://github.com/mdn/browser-compat-data/pull/11489)

**Review by foolip — APPROVED** [review]

**Comment by foolip** [comment]
@vinyldarkscratch do you want to check on the issues you raised?

**Comment by foolip** [comment]
A PR this large is prone to get conflicts, so I'll give this a quick final review and then merge.

**Comment by AdaRoseCannon** [comment]
Thank you everyone for reviewing and merging!
